tests/kuttl/test-suites/default/watcher-api-scaling/01-assert.yaml

```diff
@@ -328,6 +336,76 @@ metadata:
 spec:
   serviceName: watcher
 ---
+apiVersion: watcher.openstack.org/v1beta1
+kind: WatcherDecisionEngine
+metadata:
+  finalizers:
+  - openstack.org/watcherdecisionengine
+  name: watcher-kuttl-decisionengine
+spec:
+  containerImage: quay.io/podified-master-centos9/openstack-watcher-decision-engine:current-podified
+  memcachedInstance: memcached
+  passwordSelectors:
+    service: WatcherPassword
+  preserveJobs: false
+  replicas: 1
+  resources: {}
+  secret: watcher-kuttl
+  serviceAccount: watcher-watcher-kuttl
+  serviceUser: watcher
+status:
+  conditions:
+  - message: Setup complete
+    reason: Ready
+    status: "True"
+    type: Ready
+  - message: Deployment completed
+    reason: Ready
+    status: "True"
+    type: DeploymentReady
+  - message: Input data complete
+    reason: Ready
+    status: "True"
+    type: InputReady
+  - message: " Memcached instance has been provisioned"
+    reason: Ready
+    status: "True"
+    type: MemcachedReady
+  - message: Service config create completed
+    reason: Ready
+    status: "True"
+    type: ServiceConfigReady
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: watcher-kuttl-decisionengine
```

Comment:
`name: watcher-kuttl-decision-engine`

https://logserver.rdoproject.org/65/65/fe4aa769b44d1c038d93f638a11a82da2454d52c/github-check/watcher-operator-kuttl/968e37b/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/watcher-kuttl-default/statefulset/watcher-kuttl-decision-engine.yaml

Reply:
yes thanks updating all and rebasing